solutions/0000-0099/0001-two-sum.md

```diff
@@ -206,4 +206,22 @@ class Solution {
 }
 ```
 </TabItem>
+
+<TabItem value="javascript" label="JavaScript">
+<SolutionAuthor name="@MithunPrabhu777"/>
+
+```javascript
+var twoSum = function(nums, target) {
+    let hashMap = {};
+    
+    for(let i=0; i < nums.length; i++){
+        let difference = target - nums[i];
+        
+        if(hashMap[difference] >= 0) return [hashMap[difference],i];
+        
+        hashMap[nums[i]] = i;
+    }
+};
```

Comment:
```js
var twoSum = function(nums, target) {
    const hashMap = {};
    
    for (let i = 0; i < nums.length; i++) {
        const difference = target - nums[i];

        if (hashMap[difference] >= 0) return [hashMap[difference], i];

        hashMap[nums[i]] = i;
    }
};
```

Reply:
Okay Need to you ES6 syntax.

Let me update and push again.